functions/Test-DbaLastBackup.ps1

```diff
@@ -259,7 +268,37 @@ Restores data and log files to alternative locations and only restores databases
 					}
 					
 					$lastbackup = Get-DbaBackupHistory -SqlServer $sourceserver -Databases $dbname -LastFull -IgnoreCopyOnly:$ignorecopyonly
```

Comment:
Mmmm, I think this may be an old copy of the command? I'm not seeing this:

https://github.com/sqlcollaborative/dbatools/blob/development/functions/Test-DbaLastBackup.ps1#L316

I added some important functionality that tests the entire chain from the last log backwards, instead of just the last full.

Unfortuantely, I think that you'll have to sync that one, then paste in your changes - shouldn't be too bad, but it's kind of a pain.